{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":2,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/authfoundation\/keychainerror\/errordescription-8iygp"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/KeychainError\/errorDescription-8iygp","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"LocalizedError.errorDescription"},{"type":"text","text":"."}],"kind":"symbol","metadata":{"role":"symbol","title":"errorDescription","roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"symbolKind":"property","externalID":"s:14AuthFoundation13KeychainErrorO16errorDescriptionSSSgvp","extendedModule":"AuthFoundation","modules":[{"name":"AuthFoundation"}]},"hierarchy":{"paths":[["doc:\/\/AuthFoundation\/documentation\/AuthFoundation","doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/KeychainError","doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/KeychainError\/LocalizedError-Implementations"]]},"references":{"doc://AuthFoundation/documentation/AuthFoundation/KeychainError/LocalizedError-Implementations":{"role":"collectionGroup","title":"LocalizedError Implementations","abstract":[],"identifier":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/KeychainError\/LocalizedError-Implementations","kind":"article","type":"topic","url":"\/documentation\/authfoundation\/keychainerror\/localizederror-implementations"},"doc://AuthFoundation/documentation/AuthFoundation":{"role":"collection","title":"AuthFoundation","abstract":[{"type":"text","text":"Library that provides foundational features used by Oktaâ€™s authentication libraries, as well as APIs used to work with tokens and user credentials."}],"identifier":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation","kind":"symbol","type":"topic","url":"\/documentation\/authfoundation"},"doc://AuthFoundation/documentation/AuthFoundation/KeychainError":{"role":"symbol","title":"KeychainError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"KeychainError"}],"abstract":[],"identifier":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/KeychainError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"KeychainError"}],"url":"\/documentation\/authfoundation\/keychainerror"},"doc://AuthFoundation/documentation/AuthFoundation/KeychainError/errorDescription-8iygp":{"role":"symbol","title":"errorDescription","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"identifier":"doc:\/\/AuthFoundation\/documentation\/AuthFoundation\/KeychainError\/errorDescription-8iygp","kind":"symbol","type":"topic","url":"\/documentation\/authfoundation\/keychainerror\/errordescription-8iygp"}}}